test/parallel/test-process-cpuUsage.js

```diff
@@ -33,27 +33,57 @@ for (let i = 0; i < 10; i++) {
   assert(diffUsage.system >= 0);
 }
 
+const invalidUserArgument = new RegExp('^TypeError: value of user ' +
```

Comment:
@gibfahn thanks, I think is worth it to have a basic one and will  work in something like that for a PR